src/frontend/csrs-portal/src/app/components/questionnaire/questionnaire.component.ts

```diff
@@ -477,12 +490,31 @@ export class QuestionnaireComponent implements AfterViewInit {
     },
   ];
 
-  constructor() {
-    this.bceIdLink = 'https://www.bceid.ca/';
-    this.bceIdRegisterLink = 'https://www.bceid.ca/register/basic/account_details.aspx?type=regular&eServiceType=basic';
+  constructor(@Inject(LoggerService) private logger,
+              @Inject(Router) private router,
+              @Inject(ActivatedRoute) private route,
+              @Inject(OidcSecurityService) private oidcSecurityService) {
+
+          this.bceIdLink = 'https://www.bceid.ca/';
```

Comment:
Not sure right away what is the purposes of these links? Should those vary based on different environments you think?

Reply:
Completely agree with you! I don't know purpose of this link this.bceIdLink = 'https://www.bceid.ca/'; at all. I removed it.

Reply:
About different environments could you please provide details what should be developed.